src/systems/filecoin_blockchain/blockchain/block.id

```diff
@@ -45,6 +46,8 @@ type Chain struct {
     HeadEpoch()                 Epoch
     FinalizedEpoch()            Epoch
     LatestCheckpoint()          Epoch
+    AddTipset(ts Tipset)        
 
+    TicketAtEpoch(epoch Epoch)  Ticket
```

Comment:
this is also defined in Storage Power Consensus Subsystem https://github.com/filecoin-project/specs/blob/7e4fde83c7699f8ec33a4c8735768e4bc0bbc6fd/src/systems/filecoin_blockchain/storage_power_consensus/storage_power_consensus_subsystem.id#L27

We should remove one or the other.

Reply:
still working on that one but yes!